### PR TITLE
FEAT: Add docstring check to CI for an initial subset files

### PR DESCRIPTION
**Pull request by xmnlab**
In this PR:

- Fixed omniscidb docstrings
- Fixed some docstrings
- Added pydocstyle check for ./ibis/.py and ./ibis/omniscidb/*.py

Current issues with pydocstyle:

- https://github.com/PyCQA/pydocstyle/issues/111
- https://github.com/PyCQA/pydocstyle/issues/363

### CODE REVIEW

**Comment by xmnlab** [comment]
I will try each week to fix docstring for some backend.

**Comment by toryhaavik** [comment]
The tests are failing with ```
/home/vsts/work/_temp/b85e2339-eab9-4da6-8e4c-3f49be4fa0ab.sh: line 1: syntax error near unexpected token `('
/home/vsts/work/_temp/b85e2339-eab9-4da6-8e4c-3f49be4fa0ab.sh: line 1: `docker-compose run ibis pydocstyle --match-dir=(ibis|omniscidb)'```

**Comment by xmnlab** [comment]
thanks @toryhaavik I am working on that. it is missing the quotes. thanks!

**Comment by xmnlab** [comment]
all the tests passed now :)

File: ibis/omniscidb/tests/conftest.py [hunk 1]
```diff
@@ -17,6 +20,12 @@
 
 @pytest.fixture(scope='module')
 def con():
+    """Define a connection fixture.
```
Comment (toryhaavik) [inline ibis/omniscidb/tests/conftest.py]: we may want to consider loosening the restriction for test code. you shouldn't necessarily need to write a docstring for a test method

Comment (xmnlab) [reply ibis/omniscidb/tests/conftest.py]: IIRC the default for `match` parameter exclude tests for the checking. I just add that to help people to understand easily (as much as possible)  the tests. not sure if it is the right direction ... 

also there is an issue related to the documentation and tests experiences: https://github.com/ibis-project/ibis/issues/1989

**Comment by xmnlab** [comment]
thanks for the review and the merge @toryhaavik !

**Pull request merged**